lib/allocate.dl

```diff
@@ -1,19 +1,21 @@
-// Allocate a unique 64-bit identifier in the range `1..max_val` for each element in `toallocate`.
+// Allocate a unique identifier in the range `min_val..max_val` for each element in `toallocate`.
```

Comment:
there is some constraint on the type 'N. What is it?

Reply:
ack